metashare/api/serializers.py

```diff
@@ -237,6 +239,12 @@ class CreatePrSerializer(serializers.Serializer):
     notes = serializers.CharField(allow_blank=True)
 
 
+class ReviewSerializer(serializers.Serializer):
+    notes = serializers.CharField(allow_blank=True)
+    status = serializers.ChoiceField(choices=["APPROVE", "REQUEST_CHANGES", "COMMENT"])
```

Comment:
@wlonk I'm guessing these are determined by what the GitHub API expects? I think as currently designed, we'll only be using `APPROVE` and `REQUEST_CHANGES`.

Reply:
They are based on the GitHub API, yes. We can take out `"COMMENT"` then.